pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
please add a newline to the last line of the file. (this should be configurable in the editor/IDE)

Reply:
Added. (I should've figured vscode has a setting for that - thanks)

On a side note, both `invenio-imoox` and `invenio-records-marc21` are missing this newline too...